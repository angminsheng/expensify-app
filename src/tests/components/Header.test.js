import React from "react";
import { shallow } from "enzyme";
import Header from "../../components/Header";
import toJSON from "enzyme-to-json";

describe("Component: Header", () => {
  test("should render header component correctly", () => {
    const wrapper = shallow(<Header />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
