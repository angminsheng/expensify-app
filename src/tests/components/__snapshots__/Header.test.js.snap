// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Header should render header component correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/secret"
  >
    Secret page
  </NavLink>
</header>
`;
